src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisClientLibConfiguration.java

```diff
@@ -276,6 +278,7 @@ public KinesisClientLibConfiguration(String applicationName,
     public KinesisClientLibConfiguration(String applicationName,
             String streamName,
             String kinesisEndpoint,
+            String dynamoDBEndpoint,
```

Comment:
@pfifer I think I created the backwards compatibility you were asking for. I created another constructor that matches the old signature. This delegates to the new constructor with the dynamoDBEndpoint addition.